src/SFA.DAS.Data.DatabaseTests/StepBindings/InsertStepBindings.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BoDi;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+using System.Collections.Generic;
+using SFA.DAS.Data.DatabaseTests.TestHelpers;
+using System;
+
+namespace SFA.DAS.Data.DatabaseTests.StepBindings
+{
+    [Binding]
+    public class InsertStepBindings
+    {
+        private InsertHelper dbHelper;
+
+        public InsertStepBindings()
+        {
+            dbHelper = new InsertHelper(ScenarioContext.Current.Get<string>("connectionstring"));
+        }
+
+        private void Insertinto(IEnumerable<dynamic> values, Action<dynamic> action)
+        {
+            foreach (dynamic value in values)
+            {
+                action(value);
+            }
+        }
+
+        [Given(@"I have DAS_LevyDeclarations")]
```

Comment:
@MichaelYoung1981 when we discussed these tests a while back, we were leaning towards testing at the events boundary, rather than starting in the RDS. That way we'd also provide test coverage for the C# parts of the `data` solution. What do you think?